### DISCUSSION TITLE
Unable to launch NVDA from runnvda.bat

### DISCUSSION THREAD

**Discussion by noorkaif97** [nvaccess/nvda#15296] [Q&A]
Hi Team,

I have cloned the repo and then I tried to launch nvda from runnvda.bat command but nothing happened. Could you please help me resolve this issue. Attaching the command prompt screenshot for reference.

![image](https://github.com/nvaccess/nvda/assets/41202038/0a8500d7-7f55-4a91-8a04-2c7c9c0162f6)

**Comment by seanbudd** [comment]
Welcome @noorkaif97 ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- Many developers, contributors and community members here are vision impaired. Screenshots are not accessible. Provide an alternative description, and capture any relevant information from images.
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
Do you have python 3.7 installed correctly? Your default python installation appears to be 3.11

**Reply by noorkaif97** [reply]
Hi Thank you so much for your response.
I even tried with python 3.7 error I am still unable to launch nvda from the batch file. I am a beginner here if there is any other documentation or user guide available can you please share. Below is the error I am getting when running with python 3.7

call cmd /k  C:\Users\noora\AppData\Local\Programs\Python\Python37-32\python.exe "C:\Users\noora\nvda\source\nvda.pyw"
Traceback (most recent call last):
  File "C:\Users\noora\nvda\source\nvda.pyw", line 63, in <module>
    import config
  File "C:\Users\noora\nvda\source\config\__init__.py", line 36, in <module>
    from . import profileUpgrader
  File "C:\Users\noora\nvda\source\config\profileUpgrader.py", line 11, in <module>
    from . import profileUpgradeSteps
  File "C:\Users\noora\nvda\source\config\profileUpgradeSteps.py", line 18, in <module>
    from config.configFlags import (
  File "C:\Users\noora\nvda\source\config\configFlags.py", line 23, in <module>
    class NVDAKey(DisplayStringIntFlag):
  File "C:\Users\noora\AppData\Local\Programs\Python\Python37-32\lib\abc.py", line 126, in __new__
    cls = super().__new__(mcls, name, bases, namespace, **kwargs)
  File "C:\Users\noora\AppData\Local\Programs\Python\Python37-32\lib\enum.py", line 211, in __new__
    enum_member = __new__(enum_class)
TypeError: object.__new__(NVDAKey) is not safe, use int.__new__()

**Reply by seanbudd** [reply]
Is this log from `nvda.log` found in `C:\Users\noora\nvda`?
`call cmd /k C:\Users\noora\AppData\Local\Programs\Python\Python37-32\python.exe "` is not the correct way to start NVDA from source, you should always use `runnvda.bat`

**Reply by noorkaif97** [reply]
This is the output from runnvda.bat I changed the command a bit to use python 37 version

**Reply by seanbudd** [reply]
What version of python 3.7 are you using?

Can you produce a debug log file using the steps outlined in an earlier comment?

**Reply by noorkaif97** [reply]
No There is not nvda.log file available in the repo.
I am using python 3.7.0 to execute the code

**Reply by seanbudd** [reply]
Please update your python to 3.7.9 at the very least

https://www.python.org/downloads/release/python-379/

**Reply by noorkaif97** [reply]
Worked with this version. Thank you for your help.
